src/ProjectNeon/Assets/Features/Battle/Cards/CardResolutionZone.cs

```diff
@@ -11,4 +11,11 @@ public void Add(PlayedCard played)
     {
         moves.Add(played);
     }
+
+    public void resolve()
```

Comment:
Public methods must be PascalCase. 

https://github.com/EnigmaDragons/ProjectNeon/blob/master/guides/pr-review-policy.md